react/src/objects/stack/SprkStack.js

```diff
@@ -30,6 +31,7 @@ const SprkStack = (props) => {
     'sprk-o-Stack--split@m': splitAt === 'medium',
     'sprk-o-Stack--split@l': splitAt === 'large',
     'sprk-o-Stack--split@xl': splitAt === 'huge',
+    'sprk-o-Stack__item': isStackItem,
```

Comment:
I had added this isStackItem prop to angular as part of masthead and planned on adding it to react when doing the react masthead but the time has come for it to be added now. So we will ship this new feature for it. I also added the test for this in the test file.